### PR TITLE
Gql  api tests nazarenko

### PR DESCRIPTION
**Pull request by NazarenkoDenis37**

### CODE REVIEW

File: src/modules/user/_test/userCreate.specNaz.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      const postData = {
+        query: userCreateM,
+        variables: arg
+      }
+      requestGql(postData)
+          .expect(200)
+          .end((err, res) => {
+            if (err) return done(err);
+            const respData = res.body.data;
+            console.log("RESP BODY ===", respData)
```
Comment (slborisova) [inline src/modules/user/_test/userCreate.specNaz.js:18]: Please, delete console.log

File: src/modules/user/_test/userCreate.specNaz.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      const postData = {
+        query: ``,
+        variables: arg
+      }
+      requestGql(postData)
+          .expect(400)
+          .end((err, res) => {
+            if (err) return done(err);
+            const respData = res.body.errors[0];
+            console.log("RESP BODY ===", respData)
```
Comment (slborisova) [inline src/modules/user/_test/userCreate.specNaz.js:36]: Please, delete console.log

File: src/modules/user/_test/userCreate.specNaz.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      const postData = {
+        query: userCreateM,
+        variables: invalidInput
+      }
+      requestGql(postData)
+          .expect(400)
+          .end((err, res) => {
+            if (err) return done(err);
+            const respData = res.body.errors[0];
+            console.log("RESP BODY ===", respData)
```
Comment (slborisova) [inline src/modules/user/_test/userCreate.specNaz.js:58]: Please, delete console.log

**Review by slborisova — CHANGES_REQUESTED** [review]

File: src/modules/user/_test/userCreate.specNaz.js (hunk 1 above)
Comment (NazarenkoDenis37) [reply src/modules/user/_test/userCreate.specNaz.js:18]: done

File: src/modules/user/_test/userCreate.specNaz.js (hunk 2 above)
Comment (NazarenkoDenis37) [reply src/modules/user/_test/userCreate.specNaz.js:36]: done

File: src/modules/user/_test/userCreate.specNaz.js (hunk 3 above)
Comment (NazarenkoDenis37) [reply src/modules/user/_test/userCreate.specNaz.js:58]: done

**Review by slborisova — APPROVED** [review]

**Review by slborisova — APPROVED** [review]

**Review by slborisova — APPROVED** [review]

**Review by slborisova — APPROVED** [review]

**Review by slborisova — APPROVED** [review]

**Review by slborisova — APPROVED** [review]

**Review by slborisova — APPROVED** [review]

**Pull request merged**